plugins/generic/externalFeed/classes/ExternalFeed.inc.php

```diff
@@ -1,14 +1,14 @@
 <?php
 
 /**
- * @file plugins/generic/externalFeed/ExternalFeed.inc.php
+ * @file plugins/generic/externalFeed/classes/ExternalFeed.inc.php
  *
  * Copyright (c) 2014-2016 Simon Fraser University Library
```

Comment:
All "Simon Fraser University Library" copyrights need to be changed to "Simon Fraser University". I believe this change has already been made in the `master` branch, so a rebase ought to correct it.